docs/contributing/3.0-development/plugin-layout.rst

```diff
@@ -84,3 +84,12 @@ of a plugin app, and so on.
 
 This matching of module names is required for the Pulp Platform to be able to auto-discover
 plugin components, particularly for both models and viewsets.
+
+
+Packaging
+---------
+
+The Plugin API is available from PyPI as pulpcore-plugin. A plugin writer needs to specify the
+version of pulpcore-plugin their plugin is dependent on. A plugin writer does not need to be
+concerned which version of pulpcore would work with their plugin since pulpcore-plugin will
+resolve the pulpcore dependency.
```

Comment:
Do you think we should indicate that they can expect the pulpcore will be within the 3.y line? I think plugin writers who indirectly receive core would like to know which X version of the REST API they will receive. I'm not entirely sure how to express this. Maybe we should even leave it out. Whatever you think is best I'm good with.